docs/share.md

```diff
@@ -3,6 +3,86 @@ id: share
 title: Share
 ---
 
+## Example
+
+<div class="toggler">
+  <ul role="tablist" class="toggle-syntax">
+    <li id="functional" class="button-functional" aria-selected="false" role="tab" tabindex="0" aria-controls="functionaltab" onclick="displayTabs('syntax', 'functional')">
+      Function Component Example
+    </li>
+    <li id="classical" class="button-classical" aria-selected="false" role="tab" tabindex="0" aria-controls="classicaltab" onclick="displayTabs('syntax', 'classical')">
+      Class Component Example
+    </li>
+  </ul>
+</div>
+
+<block class="functional syntax" />
+
+```SnackPlayer name=Function%20Component%20Example&supportedPlatforms=ios,android
+import React from "react";
+import { Share, Button } from "react-native";
+
+export default const ShareExample = () => {
```

Comment:
This syntax is invalid and produce error. I have deliberate posted the right way. Please test you examples before committing.

<img width="193" alt="Annotation 2020-02-27 162156" src="https://user-images.githubusercontent.com/719641/75457891-58927a00-597d-11ea-882c-6d8e0037c381.png">